appvalidator/specs/webapps.py

```diff
@@ -267,6 +306,23 @@ class WebappSpec(Spec):
                     "navigation": {"expected_type": bool},
                 }
             },
+            # For now, we only support languages pack targeting gaia apps and
+            # only with 2.2 version.
+            "languages-target": {
+                "expected_type": dict,
+                "process": lambda s: s.process_languages_target,
+                "allowed_nodes": ["app://*.gaiamobile.org/manifest.webapp"],
+                "unknown_node_level": "error",
+                "not_empty": True,
+                "child_nodes": {
+                    "app://*.gaiamobile.org/manifest.webapp": {
+                        "expected_type": types.StringTypes,
+                        "not_empty": True,
+                        "value_matches": r"^2[.]2$"
```

Comment:
Yeah, it's intentional because we really don't know what's going to happen in the future.

I'll replace the `[.]` if you feel the `\.` is clearer.